recipes/horton/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -xe
+
+./setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Let's move this line over to `meta.yaml`, under `build.script`. I'll put another comment there.